frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -434,7 +431,31 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
       return;
     }
     await handleUpdateUser(meId, data)
-      .then(() => {
+      .then((result) => {
+        if (result.isProfileComplete) {
+          if (result.isProfileComplete) {
+            sessionStorage.setItem("currentProfileStatus", "Complete");
+          } else {
+            sessionStorage.setItem("currentProfileStatus", "InComplete");
+          }
+          const preProfileStatus = sessionStorage.getItem("preProfileStatus");
+          const currentProfileStatus = sessionStorage.getItem(
+            "currentProfileStatus",
+          );
+
+          if (preProfileStatus === "InComplete") {
```

Comment:
Could you constrain these values with enums or constants or something?  Also with your storage keys.